docs/guides/openshift/merlininstall.md

```diff
@@ -1,6 +1,28 @@
 
 This guide assumes RedHat OpenShift is running, which will allow for the install of IBM i Merlin.
 
+# Merlin Requirements
+
+IBM i Modernization Engine for Lifecycle Integration can be [installed from either OpenShift web console](docs/guides/openshift/merlininstall.md) or CLI (`oc` command).
```

Comment:
This links to the current page?

These changes are in the Hands on Tutorial. Should these changes go into the Administrator guide since there is a page there with very similar content?

Reply:
lines 4-6 are not needed.